test/git.spec.ts

```diff
@@ -4,6 +4,7 @@ import { faker } from '../src';
 import { seededTests } from './support/seededRuns';
 
 const NON_SEEDED_BASED_RUN = 5;
+const refDate = '2020-01-01T00:00:00.000Z';
```

Comment:
```suggestion

const refDate = '2020-01-01T00:00:00.000Z';
```
I would prefer an empty line here because `NON_SEEDED_BASED_RUN` isn't really related to `refDate`